tools/clang/lib/AST/NestedNameSpecifier.cpp

```diff
@@ -512,7 +512,7 @@ operator=(const NestedNameSpecifierLocBuilder &Other) {
   
   // Free our storage, if we have any.
   if (BufferCapacity) {
-    free(Buffer);
+    delete[] Buffer;
```

Comment:
Should this (and the other change on line 650) also have the `// HLSL Change: Use overridable operator delete` comment?

Reply:
Done.